### ISSUE TITLE
[FEAT]: Having a unified UI for benchmark and demo

### ISSUE THREAD

**Issue by XuhuiZhou** [sotopia-lab/sotopia-demo#28]
### Description

It would be good to have a unified place for us to place the demo and benchmark:
<img width="1301" alt="image" src="https://github.com/sotopia-lab/sotopia-demo/assets/20436061/d6288610-78a1-4714-882e-99df09cea370">
Something like this!

Here's there github repo: https://github.com/allenai/WildBench/tree/main

@Algoroxyolo

**Comment by lwaekfjlk** [comment]
Hi @XuhuiZhou thanks pointing this out. I think it definitely looks good. If we are planning to add one leaderboard for multiple pages. I believe the repo might be renamed as something like sotopia-webui instead of the current demo. cc @Jasonqi146

**Comment by XuhuiZhou** [comment]
> Hi @XuhuiZhou thanks pointing this out. I think it definitely looks good. If we are planning to add one leaderboard for multiple pages. I believe the repo might be renamed as something like sotopia-webui instead of the current demo. cc @Jasonqi146

Right, that's a good point! Something like `sotopia-space`? We might have other tabs as well in the gradio application

**Comment by XuhuiZhou** [comment]
@Algoroxyolo did you get a chance to upload the code to the github repo? We can all take a look at it if necessary

**Comment by Jasonqi146** [comment]
Sounds good! I will update the repo later in the after noon.

**Comment by Jasonqi146** [comment]
> Sounds good! I will update the repo later in the after noon.



> > Hi @XuhuiZhou thanks pointing this out. I think it definitely looks good. If we are planning to add one leaderboard for multiple pages. I believe the repo might be renamed as something like sotopia-webui instead of the current demo. cc @Jasonqi146
> 
> Right, that's a good point! Something like `sotopia-space`? We might have other tabs as well in the gradio application

Sounds good! I will update the repo later in the after noon.

**Comment by Jasonqi146** [comment]
Hi @Algoroxyolo @XuhuiZhou , I've renamed both repos. Please see https://huggingface.co/spaces/wdplx/sotopia-space as our new website.

**Comment by lwaekfjlk** [comment]
@Algoroxyolo I think we fixed the UI for agent and scenario. Free to create a PR for leaderboard.
